### PR TITLE
Update Age class to check age constraints and tests for invalid age

### LINKED ISSUE #60: MVP - Age field
update checking logic and MESSAGE_CONSTRAINTS to match intended behavior

### PR DESCRIPTION
**Pull request by sannie-beep**
fixes #60

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Age.java [hunk 1]
```diff
@@ -1,64 +1,54 @@
...
-
         Age otherAge = (Age) other;
         return value.equals(otherAge.value);
     }
-
     @Override
     public int hashCode() {
         return value.hashCode();
     }
-
```
Comment (wkxcass) [inline src/main/java/seedu/address/model/person/Age.java:63]: Perhaps these deleted blank lines can be retained for better readability~

**Review by wkxcass — COMMENTED** [review]
Just a few nits!

File: src/main/java/seedu/address/model/person/Age.java [hunk 2]
```diff
@@ -1,64 +1,54 @@
...
      * Constructs a {@code Age}.
      *
-     * @param age A valid age (numeric value).
+     * @param age A valid age.
      */
     public Age(String age) {
         requireNonNull(age);
         checkArgument(isValidAge(age), MESSAGE_CONSTRAINTS);
-        this.value = age;
+        value = age;
```
[the remark this answers is not in the document]
Comment (sannie-beep) [reply src/main/java/seedu/address/model/person/Age.java]: Oki changed

**Comment by wkxcass** [comment]
LGTM!

**Pull request merged**